schema/metrics/metrics.mapping

```diff
@@ -0,0 +1,303 @@
+{
```

Comment:
should these files have .json extension?

Reply:
will update...